seahorn/CMakeLists.txt

```diff
@@ -223,7 +223,7 @@ macro(sea_add_fuzz_executable name)
   target_link_libraries(${EXE_NAME} PRIVATE AWS::aws-c-common )
   target_link_libraries(${EXE_NAME} PRIVATE sea_fuzz)
 
-  target_compile_options(${EXE_NAME} PRIVATE -fsanitize=fuzzer,address,undefined)
+  target_compile_options(${EXE_NAME} PRIVATE -fsanitize=fuzzer-no-link,address,undefined -fprofile-instr-generate -fcoverage-mapping)
```

Comment:
why `no-link`? 

I also think this is an option for the linker. On the next line, you tell linker to link in fuzzer. 

So maybe this is just confusing...